sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -188,6 +188,27 @@ instances:
     #
     # connection_string: <CONNECTION_STRING>
 
+    ## @param dbm - boolean - optional - default: false
+    ## Set to `true` to enable the Database Monitoring beta.
+    #
+    # dbm: false
+
+    ## Configure collection of query metrics
+    #
+    query_metrics:
```

Comment:
This also enabled collection of plans. Should we make it possible for people to turn that off if they want like they can in the other checks @justiniso? 

hmmm actually I wonder if people would even want that?...since samples is turned on by default in the other checks now too